bin/logstash

```diff
@@ -46,4 +46,11 @@ fi
 . "$(cd `dirname $SOURCEPATH`/..; pwd)/bin/logstash.lib.sh"
 setup
 
-ruby_exec "${LOGSTASH_HOME}/lib/bootstrap/environment.rb" "logstash/runner.rb" "$@"
+if [ "$1" = "-V" ] || [ "$1" = "--version" ];
+then
+  LOGSTASH_VERSION_FILE="${LOGSTASH_HOME}/logstash-core/lib/logstash/version.rb"
+  LOGSTASH_VERSION=$(awk '/./{line=$0} END{print line}' $LOGSTASH_VERSION_FILE | cut -d '"' -f 2)
```

Comment:
If you just want the last line, `sed -ne '$p'` is shorter than the awk you have.

A possibly even shorter example with just sed:

```
LOGSTASH_VERSION="$(sed -ne 's/^LOGSTASH_VERSION = "\([^*]*\)"$/\1/p' logstash-core/lib/logstash/version.rb)"
```

my main concern is whether or not this works on OSX and our supported Linuxes. `awk` and `cut` may not always be installed by default (nor sed, for that matter). Thoughts?